packages/sitecore-jss-react/src/components/BYOCComponent.tsx

```diff
@@ -30,21 +30,17 @@ export type BYOCProps = {
   /**
    * rendering params
    */
-  params?: BYOCRenderingParams;
+  params?: BYOCComponentParams;
   /**
    * fields from datasource items to be passed as rendered child component props
    */
   fields?: ComponentFields;
 };
 
 /**
- * Props for BYOCRenderer component. Includes components list to load external components from.
+ * Props for BYOCComponent. Includes components list to load external components from.
  */
-export type BYOCRendererProps = BYOCProps & {
-  /**
-   * Registered component collection. Would be taken from FEAAS.External.registered
-   */
-  components: RegisteredComponents;
+export type BYOCComponentProps = BYOCProps & {
```

Comment:
Is there a reason we need the separate `BYOCProps` type? Or can it be combined with `BYOCComponentProps`?

Reply:
The [BYOCWrapper](https://github.com/Sitecore/jss/blob/1ff02cb589ce2661819aeb748bc99187ceb12fa1/packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/BYOCWrapper.tsx#L4) props are of `BYOCProps` type.

Reply:
Right, but why not use `BYOCComponentProps` for `BYOCWrapper` props? We do this for FEaaSWrapper:
https://github.com/Sitecore/jss/blob/1ff02cb589ce2661819aeb748bc99187ceb12fa1/packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/FEaaSWrapper.tsx#L12